src/Microsoft.Extensions.DependencyModel/IO/DependencyContextWriter.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Extensions.DependencyModel.IO
+{
+    public class DependencyContextWriter
```

Comment:
We can split it out somewhere else, but its nice to use same classes to generate deps file as to read it